### DISCUSSION TITLE
What to do in the Step 4?

### DISCUSSION THREAD

**Discussion by TalhaBinShahid** [skills/.github#568] [Communicate using Markdown]
I am stuck at 4th step and I need help.
I need to know that where do I have to add task lists. Should I add it into the same index.md file or Do I have to add it to any other file?

**Comment by sid7367** [comment]
1. **Change** your branch to start-markdown
2. **edit** the index.md file in that branch 
3. **commit** those changes to start-markdown branch .....

**Reply by TalhaBinShahid** [reply]
Thank you
